### PR TITLE
add phpunit tests and composer

### PR DESCRIPTION
**Pull request by honzabrecka**
It would be nice to install this useful library via composer.

### CODE REVIEW

**Comment by pwfisher** [comment]
Oh, hell yeah. Thank you for the excellent work.

File: test.php [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-
-/*
-
-php test.php --foo --bar=baz
-
-php test.php -abc
-
-php test.php arg1 arg2 arg3
-
-php test.php plain-arg --foo --bar=baz --funny="spam=eggs" --also-funny=spam=eggs \
```
Comment (pwfisher) [inline test.php:11]: You caught the edge case where the argument name has a dash, but not the case where the value has an equals sign. I've seen implementations that failed in that case.

Comment (honzabrecka) [inline test.php:11]: You're right. I made a pull request, you can check.

**Comment by honzabrecka** [comment]
Thanks. Could you please register on https://packagist.org as **pwfisher** and submit **command-line-php** package? After that, everyone will be able to install this library via composer.

**Comment by honzabrecka** [comment]
@pwfisher Any news on this?

**Comment by pwfisher** [comment]
I've registered on packagist

**Comment by honzabrecka** [comment]
Excelent!
